### ISSUE TITLE
Picking does not work with multi-volume rendering

### ISSUE THREAD

**Issue by cpinter** [Slicer/Slicer#5305]
When placing fiducials on a volume that is rendered in 3D using the multi-volume technique, the fiducials are placed incorrectly.

## Steps to reproduce

* Load CTChest
* Show volume rendering using the "VTK Multi-Volume (experimental)" technique
* Set CT-AAA preset
* Try to place fiducials on the ribs or any other solid bone
* The fiducials appear around the middle of the volume

## Expected behavior

* The fiducial is placed on the bone surface (as it does using "VTK GPU Ray Casting"

## Environment
- Slicer version: Slicer-4.11.20200930
- Operating system: Windows

**Comment by cpinter** [comment]
I will try it with the latest nightly using VTK9 as well. (First I wanted a bug to reference in other issues and source code.)

**Comment by cpinter** [comment]
The same thing happens with VTK9 and r29477.

However, rendering became much slower in VTK9 using multi-volume. Using 4.11.20200930 the CTChest dataset renders with 35 FPS, and r29477 built with VTK9 only 4 FPS. Should I add a different ticket for this?

**Comment by jcfr** [comment]
> Using 4.11.20200930 the CTChest dataset renders with 35 FPS, and r29477 built with VTK9 only 4 FPS. Should I add a different ticket for this?

Yes, let's create a dedicated issue for this and associated the `domain:vtk9` label. Ideally, updating a test in upstream VTK or having a standalone example would be nice.
